healthcare/dicomweb_instance_retrieve.go

```diff
@@ -19,13 +19,19 @@ import (
 	"context"
 	"fmt"
 	"io"
-	"io/ioutil"
+	"os"
 
 	healthcare "google.golang.org/api/healthcare/v1beta1"
 )
 
 // dicomWebRetrieveInstance retrieves a specific instance.
 func dicomWebRetrieveInstance(w io.Writer, projectID, location, datasetID, dicomStoreID, dicomWebPath string, outputFile string) error {
+        // projectID := fmt.Sprintf("my-project")
+        // location := fmt.Sprintf("us-central1")
+        // datasetID := fmt.Sprintf("my-dataset")
+        // dicomStoreID := fmt.Sprintf("my-dicom-store")
+        // dicomWebPath := fmt.Sprintf("studies/1.3.6.1.4.1.11129.5.5.1113639985/series/1.3.6.1.4.1.11129.5.5.1953511724/instances/1.3.6.1.4.1.11129.5.5.9562821369")
+        // outputFile := fmt.Sprintf("instance.dcm")
```

Comment:
No need to `fmt.Sprintf` normal strings.

Reply:
Done for all files.